src/python/pants/testutil/test_base.py

```diff
@@ -611,51 +611,56 @@ def targets(self, address_spec):
             targets.append(self.build_graph.get_target(address))
         return targets
 
-    def create_library(self, path, target_type, name, sources=None, **kwargs):
+    def create_library(
+        self,
+        path: str,
+        target_type: str,
+        name: str,
+        sources: Optional[List[str]] = None,
+        java_sources: Optional[List[str]] = None,
+        provides: Optional[str] = None,
+        dependencies: Optional[List[str]] = None,
+        requirements: Optional[str] = None,
+    ):
```

Comment:
I really wish we could enforce keyword args with this via `*`, but that'd need to be a followup.